nestkernel/recording_backend_mpi.h

```diff
@@ -47,6 +47,12 @@ Description
 The `mpi` recording backend sends collected data to a remote process
 using MPI.
 
+There are two ways to set the MPI port. If both are set, option A has precedence
+
+A)
+The address is supplied via the recording backends "mpi_address" status property.
+
+B)
 The name of the MPI port to send data to is read from a file for each
```

Comment:
The block here is actually [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) that will be [rendered nicely](https://nest-simulator.readthedocs.io/en/v3.5/models/recording_backend_mpi.html) on the documentation page. Please make this a proper enumeration.

Reply:
fixed in 4f57831